src/taipy/core/config/core_section.py

```diff
@@ -71,6 +73,9 @@ class CoreSection(UniqueSection):
     _FORCE_KEY = "force"
     _DEFAULT_FORCE = False
 
+    _CORE_VERSION_KEY = "core_version"
+    _DEFAULT_LATEST_CORE_VERSION = "3.0.0"
```

Comment:
Can't we use `taipy.core.__version__` value?
```suggestion
    _CORE_VERSION = "3.0.0"
    
```

Reply:
I'm still trying to think of a better way than hard-coding the version number into the code. Do you have any suggestions on this?

Reply:
with some refactoring, it works now :D